cfme/configure/settings.py

```diff
@@ -111,12 +111,17 @@ class Visual(Updateable):
             ('reports', Select('//select[@id="perpage_reports"]')),
         ])
 
+    startpage_form = Form(
+        fields=[
+            ('login_page', Select('//select[@id="start_page"]'))])
+
     save_button = form_buttons.FormButton("Add this Time Profile")
 
-    def __init__(self, grid_view=None, tile_view=None, list_view=None):
+    def __init__(self, grid_view=None, tile_view=None, list_view=None, login_page=None):
```

Comment:
I don't actually think you need the ```__init__``` function here at all. Or at least, you don't need to accept any parameters. The reason I say this is that when you instantiate the class, whatever you **give** it is not **set** during ```__init__```. This means that essentially your Visual object is out of date compared to the VMDB system, which is something we want to avoid. I also think that your *properties* should have **db** enabled functions too, though this is less of an issue and more of a future enhancement.